generators/common/templates/src/main/asciidoc/front-end.adoc.ejs

```diff
@@ -0,0 +1,10 @@
+<%_ if (clientFramework) { %>
```

Comment:
This template could probably be moved to `client` subgenerator, so each subgenerator contributes to the documentation.